packages/clay-sticker/README.mdx

```diff
@@ -14,30 +14,34 @@ import {
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
--   [User Icon](#user-icon)
+-   [Composing](#composing)
+    -   [Using Color](#using-color)
```

Comment:
I don't think these would be nested under "Composing". I would also rename it from "Using Color" to "Display Type"

Reply:
I think I would also drop the "Composing" name and use "Positioning". I would also place this part last since its likely the least consulted. Display type would be first